### ISSUE TITLE
CuDNNGRU层中文文档问题

### ISSUE THREAD

**Issue by GrayXu** [MoyanZitto/keras-cn#138]
CuDNNGRU层应该是不支持传参dropout的。
`keras.layers.CuDNNGRU(units, kernel_initializer='glorot_uniform', recurrent_initializer='orthogonal', bias_initializer='zeros', kernel_regularizer=None, recurrent_regularizer=None, bias_regularizer=None, activity_regularizer=None, kernel_constraint=None, recurrent_constraint=None, bias_constraint=None, return_sequences=False, return_state=False, stateful=False)`

你们的文档却显示

> dropout：0~1之间的浮点数，控制输入线性变换的神经元断开比例
